Software/bsw/gen/CanIf_PBcfg.c

```diff
@@ -210,7 +210,7 @@ CanIfTxPduCfgType CanIfTxPduCfgObj[TX_CAN_L_PDU_NUM] =
     #if(CANIF_PUBLIC_PN_SUPPORT==STD_ON)
          TRUE,                           /* If CanIfPublicPnFilterSupport is enabled or disabled */
     #endif
-    #if(CANIF_PUBLIC_READ_TX_PDU_NOTIFY_STATUS_API==STD_ON)
+    #if(CANIF_PUBLIC_READTXPDU_NOTIFY_STATUS_API==STD_ON)
```

Comment:
same as above comments: Macro's names are in UpperCase separated by a "_" character. 
we shouldn't change in configuration locally (for each team) and any changes in them, feel free to request them in cfg branches